pkg/service/types.go

```diff
@@ -13,6 +16,9 @@ const (
 	namespaceNameSeparator = "/"
 )
 
+// SyntheticServiceSuffix is a random string appended to the name of the synthetic service created for each K8s service account
+var SyntheticServiceSuffix = uuid.New().String()
```

Comment:
Why does the suffix need to be random if it is generated just once? Won't something of the sort `synthetic` as a suffix suffice?

Reply:
Zero chance of a collision with anything.

Reply:
@draychev I am still wondering why this needs to be a uuid instead of a const suffix of the form `synthetic`.

Reply:
Why do you need to worry about collisions given this is just the suffix?